integration_test/third_party_apps_data/applications/couchdb/expected_logs.yaml

```diff
@@ -0,0 +1,15 @@
+log_entries:
+- log_name: "couchdb"
+  field_matchers:
+    httpRequest.remoteIp: "127.0.0.1"
+    httpRequest.requestMethod: "GET"
+    httpRequest.responseSize: "1"
+    httpRequest.serverIp: "localhost:5984"
+    httpRequest.status: 404
```

Comment:
Last time I tried these field matchers only supported strings, might have to remove this for kokoro to work.

Reply:
updated